apps/api/src/domain/domainModelValidators/tagValidator.ts

```diff
@@ -2,34 +2,22 @@ import { InternalError } from '../../lib/errors/InternalError';
 import isStringWithNonzeroLength from '../../lib/utilities/isStringWithNonzeroLength';
 import { DTO } from '../../types/DTO';
 import { Tag } from '../models/tag/tag.entity';
-import { resourceTypes } from '../types/resourceTypes';
 import { isNullOrUndefined } from '../utilities/validation/is-null-or-undefined';
-import InvalidEntityDTOError from './errors/InvalidEntityDTOError';
-import InvalidPublicationStatusError from './errors/InvalidPublicationStatusError';
-import NullOrUndefinedResourceDTOError from './errors/NullOrUndefinedResourceDTOError';
+import InvalidTagDTOError from './errors/InvalidTagDTOError';
 import TagHasNoTextError from './errors/tag/TagHasNoTextError';
 import { DomainModelValidator } from './types/DomainModelValidator';
 import { Valid } from './Valid';
 
 const tagValidator: DomainModelValidator = (dto: unknown): Valid | InternalError => {
-    if (isNullOrUndefined(dto))
-        return new InvalidEntityDTOError(resourceTypes.tag, undefined, [
-            new NullOrUndefinedResourceDTOError(resourceTypes.tag),
-        ]);
+    if (isNullOrUndefined(dto)) return new InvalidTagDTOError(dto);
 
     const innerErrors: InternalError[] = [];
 
-    const { text, id, published } = dto as DTO<Tag>;
+    const { text, id } = dto as DTO<Tag>;
```

Comment:
I guess tags don't need a published / unpublished flag

Reply:
No. Let's not even go there.